src/tools/features-status-dump/src/main.rs

```diff
@@ -0,0 +1,31 @@
+use std::collections::HashMap;
+use std::env;
+use std::fs::File;
+use std::io::BufWriter;
+use std::path::Path;
+
+use tidy::features::{Feature, collect_lang_features, collect_lib_features};
```

Comment:
Suggestion: if this tidy logic is required by two bootstrap tools, we should pull this out into a separate shared dependency (e.g. `build_helpers` and have `tidy -> build_helpers` and `features-status-dump` -> `build_helpers`). Alternatively, some other name if `build_helpers` doesn't make sense.

Reply:
it's three tools actually, this is also used by unstable-book-gen

Reply:
I looked at this briefly, and this isn't trivial to pull out.